scripts/gulp/tasks/test-unit.ts

```diff
@@ -9,11 +9,13 @@ const argv = yargs
   .option('runInBand', {})
   .option('maxWorkers', {})
   .option('detectLeaks', {})
+  .option('coverage', { default: true })
```

Comment:
It was really annoying to run tests without coverage metrics

Reply:
I personally don't care to see by default coverage when I run tests, so I would rather have `yarn test:cover` separately. But that's my preference.